Ogitor/include/OgitorsProperty.h

```diff
@@ -572,15 +572,15 @@ namespace Ogitors
         */
         virtual void init(CBaseEditor* val)
         {
-            mValue = (unsigned long)val;
+            mValue = (uintptr_t)val;
```

Comment:
basically the pointer is till truncated here - albeit generating a warning about it now